packages/kit/src/core/dev/index.js

```diff
@@ -264,36 +273,36 @@ class Watcher extends EventEmitter {
 											str.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/>/g, '&gt;');
 
 										rendered = `<!doctype html>
-											<head>
-												<meta charset="utf-8" />
-												<meta name="viewport" content="width=device-width, initial-scale=1" />
-												<style>
-													body {
-														font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
-														color: #333;
-													}
-
-													pre {
-														background: #f4f4f4;
-														padding: 1em;
-														overflow-x: auto;
-													}
-												</style>
-											</head>
-											<h1>AMP validation failed</h1>
-
-											${result.errors
-												.map(
-													(error) => `
-												<h2>${error.severity}</h2>
-												<p>Line ${error.line}, column ${error.col}: ${error.message} (<a href="${error.specUrl}">${
-														error.code
-													}</a>)</p>
-												<pre>${escape(lines[error.line - 1])}</pre>
-											`
-												)
-												.join('\n\n')}
-										`;
+										<head>
```

Comment:
I don't think there was any need to remove a level of indentation here. If `pnpm format` automatically did it for some reason then we can just ignore it, but if you manually did it then I'd probably revert the indentation change

Reply:
I did run pnpm run format and pnpm run lint.  I'll do it again with the upcoming commit.  I think it shouldn't be changing the indentation.  In the upcoming commit, I'm adding a pair braces.  They're optional, since there's just one very long statement in the handler - `this.vite.middlewares(...)`, however the original code did have the braces.

Hopefully in my upcoming commit, it will improve the indentation.

Reply:
It's still there.  I can manually get rid of the extra unindents, and rerun format.  This could have happened accidentally on my part.